File: Linguagem C/exercicios listas/soma_impares.c
#include <stdio.h>

int main()
{
    int x,y,newx,newy,i,soma;

    printf("Digite dois numeros: \n");
    scanf("%d %d",&x,&y);

    soma = 0;

    if (x > y)
    {
        newy = x;
        newx = y;
    }
    else
    {
        newy = y;
        newx = x;
    }

    for (i = (newx + 1); i < newy; i++)
    {
        if (i % 2 != 0)
        {
            soma += i;
        }
    }

    printf("SOMA DOS IMPARES  = %d", soma);

    return 0;
}

File: Python/exercicios propostos/soma_impares.py
print("Digite dois numeros: ")
x = int(input(''))
y = int(input(''))

soma = 0

if x > y:
  troca = y
  y = x
  x = troca
  
for i in range(x+1,y):
  if i % 2 != 0:
    soma = soma + i

print(f"SOMA = {soma}")